build.gradle

```diff
@@ -173,6 +173,12 @@ private void configureAndroidLibrary(Project project) {
                 unitTests {
                     includeAndroidResources = true
                 }
+                unitTests.all {
```

Comment:
I was getting this error:
`java.lang.NoClassDefFoundError: jdk/internal/reflect/GeneratedSerializationConstructorAccessor1
`
and this fixed it.  Found at https://stackoverflow.com/questions/68065743/cannot-run-gradle-test-tasks-because-of-java-lang-noclassdeffounderror-jdk-inte